8_bit_pic.py

```diff
@@ -61,12 +61,14 @@ def __init__(self, label: str) -> None:
     def __repr__(self) -> str:
         return f"Label('{self.label}')"
 
-    def encode(self, assembler: vm_types.GenericAssembler) -> bytes:
-        assembler.symbol_table["map"][self.label] = len(assembler.byte_code)
+    def encode(self, assembler_instance: vm_types.GenericAssembler) -> bytes:  # type: ignore
```

Comment:
Another type "fix" to conform to the protocol lol ... we can change the names in the protocol at some point if you think the verbose names are too much lol

Reply:
pyright checks the types .... I was type ignore to just ignore that line ... might not even be necessary anymore since I ended up renaming all the `assembler` name func params